### PR TITLE
Lock/verify snippet

### PR DESCRIPTION
**Pull request by mantagen**
image

image

To test, replace line 76 of Decoders.elm with
```
        |> optional "verified" (list string) [ "aml-recording" ]
```

### CODE REVIEW

File: server/src/router/feedback-snippet-verify.route.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  const shouldLock = verifyAction === VERIFY_LOCK;
+
+  try {
+    const apiResponse = await searchApiService.feedbackSnippet({
+      snippetId,
+      user,
+      categories,
+      countries,
+      filterText,
+      vote: shouldLock ? true : undefined,
```
Comment (Jbarget) [inline server/src/router/feedback-snippet-verify.route.js]: when we `unlock` dont we want to undo the vote up that we did before?

File: client/src/Data/Decoders.elm [hunk 2]
```diff
@@ -73,6 +73,7 @@ matchBodyDecoder =
         |> required "url" string
         |> required "page" int
         |> required "id" string
+        |> optional "verified" (list string) [ "aml-recording" ]
```
Comment (Jbarget) [inline client/src/Data/Decoders.elm]: remove default category

File: server/src/router/feedback-snippet-verify.route.js (hunk 1 above)
Comment (mantagen) [reply server/src/router/feedback-snippet-verify.route.js]: i asked Kieran and he said nope

Comment (Jbarget) [reply server/src/router/feedback-snippet-verify.route.js]: that seems unintuitive

**Pull request merged**